### PR TITLE
Rhel support in dependency installer

### PR DESCRIPTION
**Pull request by habibayassin**
semi support for RHEL 8 because
- qt5 is broken on  rhel 8
- there isn't a specific or-tools release for rhel 8

### CODE REVIEW

**Comment by habibayassin** [comment]
@vvbandeira 
Still need to install:

- [ ] OR-tools 
- [ ] QT5

File: .github/workflows/github-actions-test-rhel-dep-install.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: Test RHEL Dependency Installation script
+on:
+  # Allows you to run this workflow manually from the Actions tab
```
Comment (vvbandeira) [inline .github/workflows/github-actions-test-rhel-dep-install.yml:3]: This file is no longer needed right?

File: .github/workflows/github-actions-test-installer.yml [hunk 2]
```diff
@@ -2,14 +2,16 @@ name: Test DependencyInstaller script
 on:
   schedule:
   - cron: "0 8 * * MON"
+  push:
+  pull_request:
```
Comment (vvbandeira) [inline .github/workflows/github-actions-test-installer.yml:6]: Remove as we don't want to trigger this on every push/PR. I suggested on other PR to trigger only when the relevant files change.

**Review by vvbandeira — CHANGES_REQUESTED** [review]

File: .github/workflows/github-actions-test-rhel-dep-install.yml (hunk 1 above)
Comment (habibayassin) [reply .github/workflows/github-actions-test-rhel-dep-install.yml:3]: yes will remove it

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**